src/css/math.less

```diff
@@ -372,5 +386,34 @@
       filter: FlipH;
       -ms-filter: "FlipH";
     }
+    &.mq-arrow-both {
+      vertical-align: text-bottom;
```

Comment:
would be nice to have some tests of this next to fractions and stuff (like `\frac{1}{2} + \overarrow{abc}`) to make sure this aligns correctly when the line box is tall